### PR TITLE
fix(doc): add doctype were missing

### PR DESCRIPTION
**Pull request by vvo**

### CODE REVIEW

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```
Comment (bobylito) [inline docs/_layouts/default.html]: Shouldn't it be uppercased? https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5/Introduction_to_HTML5

Comment (vvo) [inline docs/_layouts/default.html]: It's ok lowercase (= never have to worry about lower or uppercase so).

html5 boilerplate is ok: https://github.com/h5bp/html5-boilerplate/blob/master/dist/index.html#L1

Comment (bobylito) [inline docs/_layouts/default.html]: Old school habits I guess :)

**Comment by bobylito** [comment]
:+1:

**Pull request merged**